src/app/containers/ArticleTimestamp/shouldDisplayLastUpdatedTimestamp.js

```diff
@@ -0,0 +1,37 @@
+import { isLastRelative, isSameDay, isToday } from './helpers';
+
+export const hasBeenUpdated = ({ timeDifferenceMinutes, minutesTolerance }) => {
+  return timeDifferenceMinutes > minutesTolerance;
+};
+
+export const publishedAndUpdatedToday = ({ firstPublished, lastPublished }) =>
+  isToday(firstPublished) && isSameDay(firstPublished, lastPublished);
+
+const shouldDisplayLastUpdatedTimestamp = ({
```

Comment:
I think you can just do 
```
export default ({
  minutesTolerance,
  firstPublished,
  lastPublished,
}) => ...
```

instead of declaring the const?